Translation/ConfigBuilder.php

```diff
@@ -57,14 +104,16 @@ public static function fromConfig(Config $config)
     }
 
     /**
-    * Sets the default output format.
-    *
-    * The default output format is used when the following conditions are met:
-    *   - there is no existing file for the given domain
-    *   - you haven't forced a format
-    *
-    * @param string $format
-    */
+     * Sets the default output format.
+     *
+     * The default output format is used when the following conditions are met:
+     *   - there is no existing file for the given domain
+     *   - you haven't forced a format
+     *
+     * @param string $format
+     * @return $this
+     * @return $this
```

Comment:
Only one return is needed...

It should be a blank line before the return statement